receiver/mongodbreceiver/metadata.yaml

```
@@ -37,107 +37,107 @@ metrics:
     unit: "{operations}"
     enabled: true
     sum:
+      aggregation: cumulative
       value_type: int
       monotonic: true
-      aggregation: cumulative
     attributes: [type]
-  mongodb.collections:
+  mongodb.collection:
```

```suggestion
  mongodb.collection.count:
```
Typo on my last suggestion. To be consistent with other counts, should also have `.count`.